File: mall-sellergoods-interface/src/main/java/com/marshal/sellergoods/service/TypeTemplateService.java
package com.marshal.sellergoods.service;


import com.marshal.pojo.TbTypeTemplate;
import com.marshal.util.ResponseData;
import java.util.List;

public interface TypeTemplateService {
	public List<TbTypeTemplate> findAll();

	public ResponseData query(TbTypeTemplate condition,int pageNum,int pageSize);

	public TbTypeTemplate queryById(Long id);

	public void save(TbTypeTemplate tbTypeTemplate);

	public void delete(Long[] ids);
}

File: mall-manager-web/src/main/webapp/js/service/typeTemplateService.js
// service层,与数据库交互
app.service("typeTemplateService",function($http){
    //BASE crud
    this.query=function(pageNum,pageSize,condition){
        return $http.post("../typeTemplate/query?pageNum="+pageNum+"&pageSize="+pageSize,condition);
    }

    this.save=function (typeTemplate) {
        return $http.post("../typeTemplate/save",typeTemplate);
    }

    this.remove=function (idList) {
        return $http.get("../typeTemplate/delete?idList="+idList);
    }

    this.queryById=function (id) {
        return $http.get("../typeTemplate/queryById?id="+id);
    }
});